platform/admin/ui/src/main/webapp/js/views/configuration/Service.view.js

```diff
@@ -62,6 +62,9 @@ define([
                     configuration.close();
                 });
             }
+        },
+        serializeData: function () {
```

Comment:
I'm not sure what you mean. This is to add the displayName into the object that is being given to the template.